src/%IPM/ResourceProcessor/FileCopy.cls

```diff
@@ -142,7 +142,7 @@ Method DoCopy(tSource, tTarget, pParams)
         Set tSC = $$$ERROR($$$GeneralError, "Source file does not exists")
         Quit
       }
-      If '##class(%File).CopyFile(tSource, tTarget,'..Overlay, .return) {
```

Comment:
Community PM rewrote this class but in an incorrect way. Here if File exists, should force overlay when copying (just as HSPM does). So changing this argument to 1. Copy of Directory would be depending on '..Overlay (in the code below)